web-bundle/src/main/java/com/graphhopper/replica/ReplicaCustomSpeedsFootTagParser.java

```diff
@@ -0,0 +1,62 @@
+package com.graphhopper.replica;
+
+import com.google.common.collect.ImmutableMap;
+import com.graphhopper.customspeeds.CustomSpeedsUtils;
+import com.graphhopper.reader.ReaderWay;
+import com.graphhopper.routing.ev.EdgeIntAccess;
+import com.graphhopper.routing.ev.EncodedValueLookup;
+import com.graphhopper.routing.util.parsers.FootAverageSpeedParser;
+import com.graphhopper.util.PMap;
+
+public class ReplicaCustomSpeedsFootTagParser extends FootAverageSpeedParser {
+    private final ImmutableMap<Long, Double> osmWayIdToMaxSpeed;
+
+    // Replica-specific limit for custom walking speeds
+    public static final int FOOT_MAX_SPEED = 10;
```

Comment:
A big note here: there is no max speed defined in `FootAverageSpeedParser` like there is in the equivalent car/bike classes. I first thought we could use `MEAN_SPEED`, which is 5 km/h, but that seems a bit low (?). I'm not sure if we'll ever have a case where a customer wants to increase the walking speed on particular links, but 10 km/h (~6 mph) seemed a bit more reasonable as an upper bound

Reply:
Hmm I'm not totally opposed to `2 * FootAverageSpeedParser.MEAN_SPEED`, but in my mind it almost seems a bit less future-proofed, not more - if graphhopper changes this value behind-the-scenes, do we really want the max speed setting we're deciding on here to shift (without us necessarily realizing)? The walking limit we're deciding on here should be more of a concrete value we're choosing based on common sense ("we don't model anything faster than a light jog"), not something that scales with average walking speed IMO